pythonforandroid/archs.py

```diff
@@ -234,6 +234,12 @@ def get_env(self, with_flags_in_cc=True):
 
         env['PATH'] = environ['PATH']
 
+        # for reproducible builds
+        if 'SOURCE_DATE_EPOCH' in environ:
+            for k in 'LC_ALL TZ SOURCE_DATE_EPOCH PYTHONHASHSEED BUILD_DATE BUILD_TIME'.split():
```

Comment:
Nothing against, just curious why not:
```suggestion
            for k in ('LC_ALL', 'TZ', 'SOURCE_DATE_EPOCH', 'PYTHONHASHSEED', 'BUILD_DATE', 'BUILD_TIME'):
```

Reply:
Because (for longer lists) it's shorter :sweat_smile:

And because I miss Ruby's `%w{ foo bar baz }` and Perl's `qw{ foo bar baz }`; so I use `"foo bar baz".split()`.